src/ pages/Signup/BtnModal.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './BtnModal.scss';
+
+function BtnModal() {
+  return (
+    <>
+      <button className="btnModalOne" id="btnModals">
+        약관보기
+      </button>
+      <button className="btnModalTwo" id="btnModals">
+        약관보기
+      </button>
+      <button className="btnModalThree" id="btnModals">
+        약관보기
+      </button>
```

Comment:
이부분 구현하신 스크린샷 올려주실 수 있을까요? 파일이나 클립보드로 저장한 다음 이 창에 드래그앤드랍 or ctrl+v 하시면됩니다

Reply:
> 이부분 구현하신 스크린샷 올려주실 수 있을까요? 파일이나 클립보드로 저장한 다음 이 창에 드래그앤드랍 or ctrl+v 하시면됩니다
<img width="649" alt="스크린샷 2021-12-30 오후 6 07 26" src="https://user-images.githubusercontent.com/87845749/147737615-63f0b843-9868-4e02-b863-96702410e956.png">



이렇게 구현되었습니다.

Reply:
```suggestion
      <button className="btnModals 약관보기">
        약관보기
      </button>
      <button className="btnModals 약관보기">
        약관보기
      </button>
      <button className=" btnModals 약관보기">
        약관보기
      </button>
```
- id 셀렉터 사용할 필요 없고 오히려 지금 id 셀렉터를 여러 요소가 공유하고 있는 상황이라 잘못된거 같습니다
- 공통 스타일, 개별 스타일이 동시에 필요하다면 클래스 셀렉터를 여러개 주시면 됩니다
- 또 하나 궁금한게 저 bottom 속성을 활용해서 세로축 배열을 하는게 아니라 아래처럼 해야 하지 않을까요?
```html
<div>
  <button>이용약관 동의 (필수) </button> <button>약관보기</button>
<div>
```